README.md

```diff
@@ -111,9 +111,9 @@ await novu.trigger('<TRIGGER_NAME>',
 );
 ```
 
-## Embeddable notification center
+## Embeddable Notification Center
 
-Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
+Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an IFrame embed if you are not using React.
```

Comment:
This capitalisation is very arbitrary and it is not even used in the W3.org specs:
https://www.w3.org/TR/html4/present/frames.html#h-16.5